tests/test_error.py

```diff
@@ -87,3 +89,22 @@ def test_retry_after(self):
     def test_conflict(self):
         with pytest.raises(Conflict, match='Something something.'):
             raise Conflict('Something something.')
+
+    @pytest.mark.parametrize(
+        "exception",
+        [
+            TelegramError("test message"),
+            Unauthorized("test message"),
+            InvalidToken(),
+            NetworkError("test message"),
+            BadRequest("test message"),
+            TimedOut(),
+            ChatMigrated(1234),
+            RetryAfter(12),
+            Conflict("test message"),
+        ],
+    )
+    def test_pickling(self, exception):
+        pickled = pickle.dumps(exception)
+        unpickled = pickle.loads(pickled)
+        assert type(unpickled) == type(exception)
```

Comment:
Here you should also check that the error message and, if applicable, the additional attributes are deserialized correctly

Reply:
Sure. Then I need to create one function for each exception instead of relying on `pytest.mark.parametrize()`, as each exception has different attributes. I guess I will create a new `test_errors_pickling.py` file with a new test class then.